compiler/src/dmd/declaration.d

```diff
@@ -783,25 +783,6 @@ extern (C++) final class OverDeclaration : Declaration
         return true;
     }
 
-    Dsymbol isUnique()
```

Comment:
Is this really a semantic routine, or a property function?  I don't see any mutations being done on the object.